pyshelf/metadata/mapper.py

```diff
@@ -67,3 +67,25 @@ def create_response_property(self, name, value, immutable):
             "value": value,
             "immutable": immutable
         }
+
+    def from_response_property(self, metadata_property):
+        """
+            Maps a metadata property from a response. At
+            this poinnt it basically defaults immutable if it is
+            not there.
+
+            Args:
+                metadata_property(dict)
+        """
+        if metadata_property.get("immutable") is None:
+            metadata_property["immutable"] = False
+
+    def from_response(self, metadata):
```

Comment:
Could we instead do a deepcopy and build a new mapped structure instead of altering the structure provided?  I don't know if a mapper should alter incoming data.  Not sure if that is expected.